docs/reference/modules/scripting.asciidoc

```diff
@@ -302,12 +302,17 @@ supported operations are:
 | `mapping` |Mappings (script transform feature)
 | `search`  |Search api, Percolator api and Suggester api (e.g filters, script_fields)
 | `update`  |Update api
+| `plugin`  |Any plugin that makes use of scripts under the generic `plugin` category
 |=======================================================================
 
-The following example disables scripting for `update` and `mapping` operations,
+Plugins can also define custom operations that they use scripts for instead
+of using the generic `plugin` category. Those operations can be referred to
+in the following form: `$pluginName_$operation`.
+
+The following example disables scripting for `upnother rounddate` and `mapping` operations,
 regardless of the script source, for any engine. Scripts can still be
-executed from sandboxed languages as part of `aggregations` and `search`
-operations though, as the above defaults still get applied.
+executed from sandboxed languages as part of `aggregations`, `search`
+and `plugin` operations though, as the above defaults still get applied.
 
```

Comment:
Plugins are briefly mentioned a few lines above: 

> Plugins can also define custom operations that they use scripts for instead of using the generic `plugin` category. Those operations can be referred to in the following form: `$pluginName_$operation`.

That said I haven't explained how plugins can do it, but we don't really have reference for plugins so maybe this is enough?